langchain4j-core/src/main/java/dev/langchain4j/internal/Json.java

```diff
@@ -46,6 +47,16 @@ public interface JsonCodec {
          * @throws IOException if an I/O error occurs.
          */
         InputStream toInputStream(Object o, Class<?> type) throws IOException;
+
+        /**
+         * Convert the given JSON string to an object of the given type.
+         * @param json
+         * @param type
+         * @return the object.
+         * @param <T>
```

Comment:
Looks code doesn't have javadocs where parameters are present without description. Can we remove them or provide description?
nit: extra space after javadoc

Reply:
Done